stock_request_picking_type/models/stock_request_order.py

```diff
@@ -24,6 +25,7 @@ def _get_default_picking_type(self):
                             False,
                         ],
                     ),
+                    ("warehouse_id", "=", False),
```

Comment:
Would you please provide explanation for this change?

Reply:
If multi-warehouse configuration setting is not enabled, the stock.picking.type is created with no warehouse_id. If the user does not have multi-warehouse enabled, this change will allow the default picking_type_id to be assigned properly